server/server.js

```diff
@@ -21,35 +22,50 @@ app.use(bodyParser.urlencoded({
 app.use('/static', express.static(path.join(__dirname, '/static')));
 
 
+
+var bobList = [];
+
+
+function nowPlusXMinutes(numMinutes) {
+  return new Date(Date.now + numMinutes*60000);
+}
+
+
 // Main board
+
+// Define routes
 app.get('/', function (req, res) {
```

Comment:
need to think about how we want to organize this (similar to the controller routes? Is it ok if only the base route is present here?)